change/@fluentui-eslint-plugin-d93597a8-648b-44c7-ac16-915198c4cf6f.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

these changes have no effect on production - lets use none to not trigger any release.

> Please apply to all changefiles

```suggestion
  "type": "none",
```